core/trino-main/src/main/java/io/trino/sql/planner/plan/PatternRecognitionNode.java

```diff
@@ -138,20 +145,21 @@ public PatternRecognitionNode(
     public List<Symbol> getOutputSymbols()
     {
         ImmutableList.Builder<Symbol> outputSymbols = ImmutableList.builder();
-        if (rowsPerMatch.isOneRow()) {
+        if (rowsPerMatch == ONE) {
```

Comment:
Why did this change?

Reply:
`isOneRow()` indicates that the operation returns one output row for an input row. It applies to both `ONE ROW PER MATCH` and pattern recognition in window. 
However, those two cases differ in what columns are returned: `ONE ROW PER MATCH` is special in that it doesn't return all input columns, but only the partitioning columns. Window returns all input columns.
So, this condition had to change, because it unnecessarily caught window.
(This was not a bug though, because window wasn't supported at this stage).